presidio-anonymizer/tests/integration/test_deanonymize_engine.py

```diff
@@ -20,7 +20,7 @@ def test_given_operator_decrypt_with_valid_params_then_decrypt_text_successfully
     decryption = engine.deanonymize(
         text,
         encryption_results,
-        {"DEFAULT": OperatorConfig(Decrypt.NAME, {"key": "WmZq4t7w!z%C&F)J"})},
+        {"DEFAULT": OperatorConfig(Decrypt.NAME, {"key": b'WmZq4t7w!z%C&F)J'})},
```

Comment:
Does this require a change [in this notebook](https://github.com/microsoft/presidio/blob/main/docs/samples/python/encrypt_decrypt.ipynb) and in the [tutorial](https://github.com/microsoft/presidio/blob/main/docs/tutorial/12_encryption.md]?

Reply:
No, its another option, kept one test with that option, one with the string option